tendermint/src/vote/sign_vote.rs

```diff
@@ -234,34 +235,32 @@ mod tests {
         let got = request.to_signable_vec().unwrap();
 
         // the following vector is generated via:
-        /*
-           import (
-               "fmt"
-               prototypes "github.com/tendermint/tendermint/proto/tendermint/types"
-               "github.com/tendermint/tendermint/types"
-               "strings"
-               "time"
-           )
-           func voteSerialize() {
-               stamp, _ := time.Parse(time.RFC3339Nano, "2017-12-25T03:00:01.234Z")
-               vote := &types.Vote{
-                   Type:      prototypes.PrevoteType, // pre-vote
-                   Height:    12345,
-                   Round:     2,
-                   Timestamp: stamp,
-                   BlockID: types.BlockID{
-                       Hash: []byte(""),
-                       PartSetHeader: types.PartSetHeader{
-                           Total: 1000000,
-                           Hash:  []byte("0022446688AACCEE1133557799BBDDFF"),
-                       },
-                   },
-                   ValidatorAddress: []byte{0xa3, 0xb2, 0xcc, 0xdd, 0x71, 0x86, 0xf1, 0x68, 0x5f, 0x21,
-                       0xf2, 0x48, 0x2a, 0xf4, 0xfb, 0x34, 0x46, 0xa8, 0x4b, 0x35}, ValidatorIndex: 56789}
-               signBytes := types.VoteSignBytes("test_chain_id", vote.ToProto())
-               fmt.Println(strings.Join(strings.Split(fmt.Sprintf("%v", signBytes), " "), ", "))
-           }
-        */
+        // import (
+        // "fmt"
+        // prototypes "github.com/tendermint/tendermint/proto/tendermint/types"
+        // "github.com/tendermint/tendermint/types"
+        // "strings"
+        // "time"
+        // )
+        // func voteSerialize() {
+        // stamp, _ := time.Parse(time.RFC3339Nano, "2017-12-25T03:00:01.234Z")
+        // vote := &types.Vote{
+        // Type:      prototypes.PrevoteType, // pre-vote
+        // Height:    12345,
+        // Round:     2,
+        // Timestamp: stamp,
+        // BlockID: types.BlockID{
+        // Hash: []byte(""),
+        // PartSetHeader: types.PartSetHeader{
+        // Total: 1000000,
+        // Hash:  []byte("0022446688AACCEE1133557799BBDDFF"),
+        // },
+        // },
+        // ValidatorAddress: []byte{0xa3, 0xb2, 0xcc, 0xdd, 0x71, 0x86, 0xf1, 0x68, 0x5f, 0x21,
+        // 0xf2, 0x48, 0x2a, 0xf4, 0xfb, 0x34, 0x46, 0xa8, 0x4b, 0x35}, ValidatorIndex: 56789}
+        // signBytes := types.VoteSignBytes("test_chain_id", vote.ToProto())
+        // fmt.Println(strings.Join(strings.Split(fmt.Sprintf("%v", signBytes), " "), ", "))
+        // }
```

Comment:
Same. Is there any way to tell the formatter to ignore a particular block comment?